test/units/parsing/test_splitter.py

```diff
@@ -61,12 +80,27 @@
     (u'a="multiline\nmessage1\\\n" b="multiline\nmessage2\\\n"',
         [u'a="multiline\nmessage1\\\n"', u'b="multiline\nmessage2\\\n"'],
         {u'a': 'multiline\nmessage1\\\n', u'b': u'multiline\nmessage2\\\n'}),
+    (u'line \\\ncontinuation',
+        [u'line', u'continuation'],
+        {u'_raw_params': u'line continuation'}),
+    (u'not jinja}}',
+        [u'not', u'jinja}}'],
+        {u'_raw_params': u'not jinja}}'}),
+    (u'a={{multiline\njinja}}',
+        [u'a={{multiline\njinja}}'],
+        {u'a': u'{{multiline\njinja}}'}),
```

Comment:
This test verifies the new behavior, which I believe is correct. However, it is a change to parsing behavior. Previously this would result in each `\n` turning into `\n\n` instead.

Reply:
iirc, that was a long standing bug, that it was not 'multiline' aware